Source/CoreFramework/AppAuthCore.h

```diff
@@ -48,5 +48,6 @@ FOUNDATION_EXPORT const unsigned char AppAuthCoreVersionString[];
 #import <AppAuthCore/OIDTokenResponse.h>
 #import <AppAuthCore/OIDTokenUtilities.h>
 #import <AppAuthCore/OIDURLSessionProvider.h>
-
+#import "AppAuthCore/OIDEndSessionRequest.h"
+#import "AppAuthCore/OIDEndSessionResponse.h"
```

Comment:
Should we use the `<` import style here to match?

Reply:
Oops, my bad. I'll fix this!

Reply:
Done